cleanUp.m

```diff
@@ -12,6 +12,11 @@
 % Screen Close All
 sca;
 
+% Close Psychportaudio if open 
+if PsychPortAudio('GetOpenDeviceCount') ~= 0
+	PsychPortAudio('Close');
```

Comment:
Actually I might have merged too quickly. 

Do you think that the `if` is necessary?

Can we just not do a psychportaudio(close)

Reply:
Yes, it should work anyway. I used that because biased by macs behavior where if you close things that were not opened it crashes badly, e.g. using `sca` or `close all` at the beginning of a ptb experiment. But I could be wrong in the case of psychportaudio.

Reply:
OK let's keep it that way then. :-)